test/Token.js

```diff
@@ -53,39 +46,52 @@ describe("Token contract", function () {
 
     // If the callback function is async, Mocha will `await` it.
     it("Should set the right owner", async function () {
+      // We use loadFixture to setup our environment, and then assert that
+      // things went well
+      const { hardhatToken, owner } = await loadFixture(deployTokenFixture);
       // Expect receives a value, and wraps it in an assertion objet. These
       // objects have a lot of utility methods to assert values.
 
-      // This test expects the owner variable stored in the contract to be equal
-      // to our Signer's owner.
+      // This test expects the owner variable stored in the contract to be
+      // equal to our Signer's owner.
       expect(await hardhatToken.owner()).to.equal(owner.address);
     });
 
     it("Should assign the total supply of tokens to the owner", async function () {
+      const { hardhatToken, owner } = await loadFixture(deployTokenFixture);
       const ownerBalance = await hardhatToken.balanceOf(owner.address);
       expect(await hardhatToken.totalSupply()).to.equal(ownerBalance);
     });
   });
 
   describe("Transactions", function () {
     it("Should transfer tokens between accounts", async function () {
+      const { hardhatToken, owner, addr1, addr2 } = await loadFixture(deployTokenFixture);
       // Transfer 50 tokens from owner to addr1
-      await hardhatToken.transfer(addr1.address, 50);
-      const addr1Balance = await hardhatToken.balanceOf(
-        addr1.address
-      );
-      expect(addr1Balance).to.equal(50);
+      await expect(hardhatToken.transfer(addr1.address, 50))
+        .to.changeTokenBalances(hardhatToken, [owner, addr1], [-50, 50]);
 
       // Transfer 50 tokens from addr1 to addr2
       // We use .connect(signer) to send a transaction from another account
-      await hardhatToken.connect(addr1).transfer(addr2.address, 50);
-      const addr2Balance = await hardhatToken.balanceOf(
-        addr2.address
-      );
-      expect(addr2Balance).to.equal(50);
+      await expect(hardhatToken.connect(addr1).transfer(addr2.address, 50))
+        .to.changeTokenBalances(hardhatToken, [addr1, addr2], [-50, 50]);
+    });
+
+    it("should emit Transfer events", async function () {
+      const { hardhatToken, owner, addr1, addr2 } = await loadFixture(deployTokenFixture);
+
+      // Transfer 50 tokens from owner to addr1
+      await expect(hardhatToken.transfer(addr1.address, 50))
+        .to.emit(hardhatToken, "Transfer").withArgs(owner.address, addr1.address, 50)
+
+      // Transfer 50 tokens from addr1 to addr2
+      // We use .connect(signer) to send a transaction from another account
+      await expect(hardhatToken.connect(addr1).transfer(addr2.address, 50))
+        .to.emit(hardhatToken, "Transfer").withArgs(addr1.address, addr2.address, 50)
     });
 
     it("Should fail if sender doesn’t have enough tokens", async function () {
```

Comment:
```suggestion
    it("Should fail if sender doesn't have enough tokens", async function () {
```

I think that's not an ascii char